net-nntp/nzbget/nzbget-18.0_pre1820.ebuild

```diff
@@ -21,18 +21,17 @@ IUSE="debug gnutls ncurses parcheck ssl test zlib"
 RDEPEND="dev-libs/libxml2
 	ncurses? ( sys-libs/ncurses:0 )
 	ssl? (
-		gnutls? ( net-libs/gnutls )
+		gnutls? (
+			net-libs/gnutls
+			dev-libs/nettle
+		)
 		!gnutls? ( dev-libs/openssl:0= )
 	)
 	zlib? ( sys-libs/zlib )"
 DEPEND="${RDEPEND}
 	virtual/pkgconfig"
 DOCS=( ChangeLog README nzbget.conf )
 
-PATCHES=(
-	"${FILESDIR}"/${P}_parcheck-tests-fix.patch
-)
-
 S=${WORKDIR}/${PN}-${PV/_pre*/-testing}
 
 pkg_pretend() {
```

Comment:
please also perform these checks in `pkg_setup`. Far too many people rely on pkg_pretend to check package invariants, which it cannot guarantee. Preferably stick it in a function and call it in `pkg_pretend` and `pkg_setup`.